src/aggregator/aggregator/aggregator.go

```diff
@@ -383,6 +393,20 @@ func (agg *aggregator) Close() error {
 	}
 	agg.state = aggregatorClosed
 
+	var (
+		lastOpCompleted = time.Now()
+		currTime        time.Time
+		closeLogger     = agg.logger.With(zap.String("closing", "aggregator"))
+
+		logCloseOperation = func(op string) {
+			currTime = time.Now()
```

Comment:
nit: why does `currTime` need to be var outside this scope?

Reply:
Ah good call, this is a holdover of a previous approach